core/qbft/qbft.go

```diff
@@ -27,34 +26,42 @@ import (
 	"github.com/obolnetwork/charon/app/errors"
 )
 
+// Value defines the generic value type constraint
+// that provides an equality method.
+type Value[V any] interface {
```

Comment:
@corverroos I don't think the name of this interface represents the methods it countais. Something like Equaler is better

Reply:
Yeah, but using `V Value` for the "value" makes a lot more sense than `Equaler` for "value". In this case, this interface defines what constraints the "value" type. Which is only that it has an equal method.

Reply:
I see that this is the interface for the value we are reaching consensus for, it makes sense to have an interface called Value. I was confused because it didn't have other methods, I guess you will need to add it some other method to be able to use it, like something that returns V? In order to use the Value[v] returned by the consensus?